src/Components/BookPage/AboutBook.js

```diff
@@ -0,0 +1,318 @@
+    // Bofa
+    ageRange = '-';
+  } else if (ageMin === ageMax) {
+    // Valid but equal ages
+    ageRange = `Age ${ageMin}`;
+  } else if (ageMin === -1) {
+    // Bad ageMin
+    ageRange = `Up to ${ageMax}`;
+  } else if (ageMax === -1) {
+    // Bad ageMax
+    ageRange = `${ageMin} - 18`;
+  } else if (ageMax < ageMin) {
+    // Bad ordering
+    ageRange = `${ageMax} - ${ageMin}`;
+  } else {
+    // Happy trail
+    ageRange = `${ageMin} - ${ageMax}`;
+  }
+  return ageRange;
+};
+
+/* Date Published
+    Take in a UTC date string and return an array of [dayOfWeek, month, day, year]
+  */
+const processDate = (date) => {
+  const dateObject = new Date(date);
+  const dateArray = dateObject.toDateString().split(' ');
+
+  // Remove leading zeros from day
+  dateArray[2] = parseInt(dateArray[2], 10);
+  return dateArray;
+};
+
+function AboutBook({
+  authors, illustrators, ageMin, ageMax, gradeMin, gradeMax,
+  bookType, datePublished, bookshopURL,
+}) {
+  const gradeRange = createGradeRange(gradeMin, gradeMax);
+  const ageRange = createAgeRange(ageMin, ageMax);
+
+  // Process date_published
+  const publishDate = processDate(datePublished);
+
+  const authorLinks = authors.map((author, i) => {
+    // Add comment to links unless it is the last
+    const link = (i !== authors.length - 1) ? `${author.name}, ` : author.name;
+    return (
+      <span key={author.id}>
+        <Link style={styles.link} to={`/creator/${author.id}`}>
+          {link}
+        </Link>
+      </span>
+    );
+  });
+
+  const illustratorLinks = illustrators.map((illustrator, i) => {
+    const link = (i !== illustrators.length - 1) ? `${illustrator.name}, ` : illustrator.name;
+    return (
+      <span key={illustrator.id}>
+        <Link style={styles.link} to={`/creator/${illustrator.id}`}>
+          {link}
+        </Link>
+      </span>
+    );
+  });
+  return (
+    <Card sx={styles.sideCard}>
+      <CardContent>
+        <Box sx={styles.sideCardContainer}>
+          <Typography sx={styles.sideCardTitle}>About This Book</Typography>
+          <Box sx={styles.sideCardLinkContainer}>
+            <Typography sx={styles.creator}> Written by:&nbsp;&nbsp;</Typography>
+            {authorLinks}
+            <br />
+            <Typography sx={styles.creator}> Illustrated by:&nbsp;&nbsp;</Typography>
+            {illustratorLinks}
+          </Box>
+          <div style={styles.blockContainer}>
+            <div style={styles.block}>
+              <Typography sx={styles.bolded}>
+                {ageRange}
+              </Typography>
+              <p style={styles.sub}>Age Range</p>
+            </div>
+            {
+                    (gradeMin || gradeMax) ? (
+                      <div style={styles.block}>
+                        <Box sx={styles.bolded}>
+                          {gradeRange}
+                          {' '}
+                        </Box>
+                        <p style={styles.sub}>Reading Level</p>
```

Comment:
Can you change reading to grade level? I'll talk to the designers, I think it's really confusing to not have 'grade' specified anywhere when we're just using number prefixes 

```suggestion
                        <p style={styles.sub}>Grade Level</p>
```